site/docs/about/component-milestones.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Component Milestones
+layout: DetailTechnical
+---
+
+To help us track the state of a component, we use a set of milestones:
+
+- Legacy
+- Experimental
+- In Progress
+- RC (Release Candidate)
+- Stable
+
+Legacy, Experimental, In Progress and RC are all in `@salt-ds/lab`. Stable components are considered production ready and are in `@salt-ds/core`.
```

Comment:
we have RC in core atm?

Reply:
Hi, you don't need to review this yet. This is around a new approach and it will be shared soon.

Reply:
np, should it be drafted? :)